docs/source/support/component-statuses.rst

```diff
@@ -203,18 +203,6 @@ Developer Tools
    * - ``daml sandbox`` assistant command and documented CLI under ``daml sandbox --help``.
      - Stable
      -
-   * - Daml Sandbox in Memory (ie without the ``--sql-backend-jdbcurl`` flag)
-     - Stable
-     -
-   * - Daml Sandbox on Postgres (ie with the ``--sql-backend-jdbcurl`` flag)
-     - Stable, Deprecated
-     - 2020-12-16
-   * - Daml Sandbox Classic and associated CLIs ``daml sandbox-classic``, ``daml start --sandbox-classic``
-     - Stable, Deprecated
-     - 2020-04-09
-   * - Daml Profiler ``daml sandbox --profile-dir``
```

Comment:
I don’t think we should drop this. The profiler still works and exists, it’s just activated differently. I’d drop the command line and myabe rephrase it to `Daml Profiler in Sandbox`.

Reply:
done